src/tools/test-polymer.ts

```diff
@@ -75,7 +75,7 @@ function rework(line: string) {
                           .filter(([_, v]) => v !== undefined)
                           .map(([k]) => k);
   const expectedPaths = IterableX.from(walkDir(expectedDir))
-                            .map((f) => `./${f}` as ConvertedDocumentUrl)
+                            .map((f) => `${f}` as ConvertedDocumentFilePath)
```

Comment:
Immaterial, but style-wise it feels like, for consistency, this should be:
```ts
.map((f) => f as string as ConvertedDocumentFilePath)
```

Reply:
good catch, no longer need the template string since there's no more string modifications